src/lib/cron-tab/daily.tsx

```diff
@@ -20,6 +21,7 @@ const DailyCron: FunctionComponent<DailyCronProp> = (props) => {
     }, [])
 
     const onDayChange = (e: {target: { value: string}}) => {
+        if(props.isDisabled === true) { return }
```

Comment:
disabled prop to input box will stop triggering onchange rt? 
Do we need this check?

Reply:
No, just **disabled** attribute to input box is not enough, malicious user could remove the **disabled** attr from dev tools, thus resulting in state change,
That's why extra JS check was added.